Server/application.swift

```diff
@@ -3,5 +3,10 @@ import Nest
 
 
 public func application(request: RequestType) -> ResponseType {
-  return Response(.Ok, contentType: "text/plain", body: "Hello World")
+  switch request.method {
+  case "POST":
+    return Response(.Ok, contentType: "text/plain", body: "Hello " + request.body!)
```

Comment:
I think we should safely unwrap the body here instead of `request.body`, in the case the server isn't implemented correctly it would crash when running the tests and make it harder to determine the failure. We could instead return "Request body was empty" as an error response, then the test runner would say the contents of the body didn't match "The request body was nil".